src/CMakeLists.txt

```diff
@@ -333,13 +333,13 @@ if((APPLE) AND (PKG_CONFIG_FOUND))
   endif()
 endif()
 
-find_package(Qt5 5.6 REQUIRED COMPONENTS Core Multimedia Network OpenGL UiTools Widgets Concurrent)
+find_package(Qt5 5.7 REQUIRED COMPONENTS Core Multimedia Network OpenGL UiTools Widgets Concurrent)
 
 # This is not present before Qt 5.7:
 find_package(Qt5 COMPONENTS Gamepad QUIET)
 
-if (Qt5Core_VERSION VERSION_LESS 5.6)
-  message(FATAL_ERROR "Mudlet requires Qt 5.6 or later")
+if (Qt5Core_VERSION VERSION_LESS 5.7)
+  message(FATAL_ERROR "Mudlet requires Qt 5.7 or later")
 else()
   if (Qt5Core_VERSION VERSION_LESS 5.7)
```

Comment:
This inner test is not required as it is testing the same thing as the outer test now - and as I said I ***think*** CMake will error out before here when it cannot find a Qt of 5.7 or greater in the `find_package(Qt 5.7 REQUIRED `...`)` action before here; it might be worth a test locally on your platform to confirm it - with a temporary run inside Qt Creator with a 5.6 build - but that requires you to have one present...

Anyhow - I think you need to lose the inner test - but leave the version report to the outer `else()`.

Reply:
Just want to raise the version, don't want to do any side-tweaking (and more testing) that'll potentially break things... though if you think it works, throw in a PR?

I'm too tired right now to do much more

Reply:
Right now as you have it the pseudo-code is:

    if (Qt version less then something)
        error out with a message that Qt version is too low
    else
        if (Qt version less then exactly the same something as has already been checked)
             warn that something is not available
        else
             give information about the actual Qt version
        end
    end

The bottom line is that the warning about something is not available is unreachable and is dead code - so that that inner `if` is no longer useful...

Reply:
Okay - but being a busy bee and doing side tweaks unrelated to the focus of the PR should be optional, not mandatory...

Reply:
It is not an unrelated tweak - this PR is about raising the minimum supported Qt version to 5.7 - so the removal of dead code near to a place where the code is already being changed to give a warning about something that is not available in versions **less than** 5.7 is entirely ***within*** scope at least in my opinion.